ansible/roles_ocp_workloads/ocp4_workload_rhtr_xraylab/tasks/knative-eventing-contrib.yaml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Installing KafkaSource from KNative Eventing Contrib
+  shell: "oc apply -f https://github.com/knative/eventing-contrib/releases/download/{{ ocp4_workload_rhtr_xraylab_kafkasource }}/kafka-source.yaml"
```

Comment:
Wondering if you could use the k8s module with this.

Reply:
I could but it's a huge bunch of YAMLs that I would have to break out from the release. I'd rather not as this is a tricky piece, still under development. So sticking to this specific release in the way it is documented fells safer.

Reply:
Following a deployment failure because sometimes GitHub fails to answer, or sends an error message, I've modified this step to include the definition files directly.